File: control-assurance-sp/src/types/NAOPublicationDirectorate.ts
import { IEntity } from "./Entity";

export interface INAOPublicationDirectorate extends IEntity {
    NAOPublicationId?: number;
    DirectorateId?: number;
}

export class NAOPublicationDirectorate implements INAOPublicationDirectorate {
    public ID = 0;
    public Title = null;
    public NAOPublicationId = null;
    public DirectorateId = null;
}

File: ControlAssuranceAPI/Models/NAOPublicationDirectorate.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class NAOPublicationDirectorate
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public int? NAOPublicationId { get; set; }

    public int? DirectorateId { get; set; }

    public virtual Directorate? Directorate { get; set; }

    public virtual NAOPublication? NAOPublication { get; set; }
}
